tests/compose/test_column_transformer.py

```diff
@@ -39,3 +42,45 @@ def test_column_transformer():
     expected = clone(a).fit_transform(df)
     expected = pd.DataFrame(expected, index=result.index, columns=result.columns)
     dd.utils.assert_eq(result, expected)
+
+
+def test_column_transformer_unk_chunksize():
+    cols = 3
+    rows = 4
+    a = 0x61  # character code for 'a'.
+    names = list(map(chr, a + np.arange(cols)))
+    x = db.from_sequence(gen_ints(rows, cols)).to_dataframe(columns=names)
```

Comment:
This seems a bit complicated for generating a DataFrame. How about

```python
dd.from_pandas(pd.DataFrame(np.arange(12).reshape(4, 3), columns=names), 2)
```

you'll need to adjust the expected a bit.

Reply:
Sure.  NP.  Will do.

Reply:
@TomAugspurger: Updated.  Much simpler now.